File: src/app/brain/386ed56e-6745-4378-a553-9fdfbaf72850/scratch/count_braces.js

const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\mohit\\OneDrive\\Desktop\\computer\\src\\components\\atc\\StudentManager.tsx', 'utf8');

const openBraces = (content.match(/{/g) || []).length;
const closeBraces = (content.match(/}/g) || []).length;
const openDivs = (content.match(/<div/g) || []).length;
const closeDivs = (content.match(/<\/div/g) || []).length;
const openParens = (content.match(/\(/g) || []).length;
const closeParens = (content.match(/\)/g) || []).length;

console.log(`Braces: ${openBraces} / ${closeBraces}`);
console.log(`Divs: ${openDivs} / ${closeDivs}`);
console.log(`Parens: ${openParens} / ${closeParens}`);

File: src/app/brain/386ed56e-6745-4378-a553-9fdfbaf72850/scratch/count_braces.py

with open(r'c:\Users\mohit\OneDrive\Desktop\computer\src\components\atc\StudentManager.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

open_braces = content.count('{')
close_braces = content.count('}')
open_divs = content.count('<div')
close_divs = content.count('</div')
open_parens = content.count('(')
close_parens = content.count(')')

print(f"Braces: {open_braces} / {close_braces}")
print(f"Divs: {open_divs} / {close_divs}")
print(f"Parens: {open_parens} / {close_parens}")
